pkgs/by-name/go/got/package.nix

```diff
@@ -1,44 +1,51 @@
-{ lib
-, stdenv
-, fetchurl
-, pkg-config
-, libressl
-, libbsd
-, libevent
-, libuuid
-, libossp_uuid
-, libmd
-, zlib
-, ncurses
-, bison
-, autoPatchelfHook
-, testers
-, signify
-, overrideSDK
-, withSsh ? true, openssh
-# Default editor to use when neither VISUAL nor EDITOR are defined
-, defaultEditor ? null
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkg-config,
+  libressl,
+  libbsd,
+  libevent,
+  libuuid,
+  libossp_uuid,
+  libmd,
+  zlib,
+  ncurses,
+  bison,
+  autoPatchelfHook,
+  testers,
+  signify,
+  apple-sdk_15,
+  nix-update-script,
+  withSsh ? true,
+  openssh,
+  # Default editor to use when neither VISUAL nor EDITOR are defined
+  defaultEditor ? null,
 }:
 
-let
-  stdenv' = if stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isx86_64
-    then overrideSDK stdenv "11.0"
-    else stdenv;
-in
-stdenv'.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs: {
   pname = "got";
-  version = "0.106";
+  version = "0.108";
 
   src = fetchurl {
     url = "https://gameoftrees.org/releases/portable/got-portable-${finalAttrs.version}.tar.gz";
-    hash = "sha256-MHnXQsElBH3jOd2SPXXQuWCZWjpLVn7QjvNtESvbB8w=";
+    hash = "sha256-bI0yCt01h65HwG2jLUH+5Ah+1mGTG6ZNxt53v0Mr0+I=";
   };
 
-  nativeBuildInputs = [ pkg-config bison ]
-    ++ lib.optionals stdenv.hostPlatform.isLinux [ autoPatchelfHook ];
+  nativeBuildInputs = [
+    pkg-config
+    bison
+  ] ++ lib.optionals stdenv.hostPlatform.isLinux [ autoPatchelfHook ];
 
-  buildInputs = [ libressl libbsd libevent libuuid libmd zlib ncurses ]
-    ++ lib.optionals stdenv.hostPlatform.isDarwin [ libossp_uuid ];
+  buildInputs = [
+    libressl
+    libbsd
+    libevent
+    libuuid
+    libmd
+    zlib
+    ncurses
+  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [ libossp_uuid apple-sdk_15 ];
```

Comment:
Is this invariant under nixfmt?

Reply:
No, it wasn't. Thanks for catching that 👍